packages/common/src/common.ts

```diff
@@ -66,8 +66,9 @@ export class Common {
     // Assign hardfork changes in the sequence of the applied hardforks
     this.HARDFORK_CHANGES = this.hardforks().map((hf) => [
       hf.name,
-      hardforksDict[hf.name] ??
-        (this._chainParams.customHardforks && this._chainParams.customHardforks[hf.name]),
+      // Allow to even override an existing hardfork specification
+      (this._chainParams.customHardforks && this._chainParams.customHardforks[hf.name]) ?? 
+      hardforksDict[hf.name],
```

Comment:
important change here cc @jochem-brouwer @acolytec3 @holgerd77 

(please see comment)

Reply:
Ok, this works, but the problem is that if you want to correctly override it, you first have to import all the params from the original hardfork and then set that as custom hardfork

Reply:
My initial thoughts here are: load the original `hardforksDict[hf.name]` and then merge the overridden changes in it? (via copy so via JSON.stringify / JSON.parse)